public/js/actions/chartsActions.js

```diff
@@ -1,33 +1,106 @@
-export const updateComposerVsIncopy = ({ chartData, startDate, endDate }) => ({
-    type: 'UPDATE_COMPOSER_VS_INCOPY',
-    chartData,
-    startDate,
-    endDate
-});
+import moment from 'moment';
```

Comment:
The changes in `chartActions` and `chartsReducer` is moving business logic from one place to the other. Mainly to make sure that `moment` dates don't make it into the reducer and potentially get stored. Also, while it doesn't especially matter in the app at present, it's best practice for async reasons: https://github.com/reactjs/redux/issues/1171